packages/odyssey-react/src/components/Text/Text.theme.ts

```diff
@@ -26,12 +26,14 @@ export const theme: ThemeReducer = (theme) => ({
 
   LineHeight: theme.FontLineHeightBody,
   BodyLineHeight: theme.FontLineHeightBody,
+  /* eslint-disable @okta/odyssey/no-invalid-theme-properties */
   Heading1LineHeight: theme.FontLineHeightHeading1,
   Heading2LineHeight: theme.FontLineHeightHeading2,
   Heading3LineHeight: theme.FontLineHeightHeading3,
   Heading4LineHeight: theme.FontLineHeightHeading4,
   Heading5LineHeight: theme.FontLineHeightHeading5,
   Heading6LineHeight: theme.FontLineHeightHeading6,
```

Comment:
cc @edburyenegren-okta these tokens violate our numbers should only appear as suffix rule but not gonna worry too much for now

Reply:
I thought we were changing the rule specifically to allow mid-token numbers so the modifier matches the token format.

Ex: Line 31 is the `line-height` property for `Heading3`, not the third variation of `Heading` `line-height` (as in `HeadingLineHeight3`).

Reply:
Ugh, maybe our wires got crossed at some point. At the time, I thought these were still escaped/disallowed, which is why we had `Remove any unnecessary rule escapes` as one of the AC. 😓